packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -54,12 +54,12 @@ function toTimestamp(date) {
     return timestamp
   }
 
-  const ms = parseDateTime(date)
-  if (!ms) {
+  const ms = parseDateTime(date).getTime()
```

Comment:
What if `date` does not respect the expected format (`%Y%m%dT%H:%M:%SZ`)?

Reply:
We have problem !
`Cannot read property 'getTime' of null`

Reply:
We need to handle this.

Reply:
I just modified it. Better ?

Reply:
Yep, at least it does not break anymore :slightly_smiling_face: